app/components/UI/Navbar/index.js

```diff
@@ -682,17 +682,61 @@ export function getOnboardingNavbarOptions(
  *
  * @returns {Object} - Corresponding navbar options containing headerTitle
  */
-export function getTransparentOnboardingNavbarOptions(themeColors) {
+export function getTransparentOnboardingNavbarOptions(
```

Comment:
Am I missing something or this function got changed, but it is not used anymore?